### PR TITLE
Create Godot.gitignore

### PR DESCRIPTION
**Pull request by fahall**
A simple .gitignore for the Godot game engine.

 - **Project’s homepage**: [The Godot Engine](www.godotengine.org)
 - **Template**: [.gitignore example from godot demo project.](https://github.com/godotengine/godot-demo-projects/blob/master/.gitignore)

### CODE REVIEW

**Comment by aBARICHELLO** [comment]
Why ignore the .import files? They are important while cloning a project to prevent things like all PNG textures having filter on.

**Comment by shiftkey** [comment]
@BastiaanOlij @akien-mga (who wrote the linked gitignore) - anything to add here?

**Comment by akien-mga** [comment]
Seems fine, though should `# System/tool-specific ignores` be part of the Godot .gitignore? Those are not Godot-related, just standard files produced by users' file managers and IDEs.

**Comment by shiftkey** [comment]
@akien-mga good point, and yes, this should be Godot-specific rules.

File: Godot.gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Godot-specific ignores
+.import/
+export.cfg
+export_presets.cfg
+
+# Mono-specific ignores
+.mono/
+
+# System/tool-specific ignores
+.directory
```
Comment (shiftkey) [inline Godot.gitignore]: This isn't specific to Godot - could you please remove it?

**Comment by shiftkey** [comment]
@fahall thanks!

**Pull request merged**